workshops/login_auth/README.md

```diff
@@ -0,0 +1,540 @@
+<script src="script.js"></script>
+</body>
+</html>
+```
+
+#### JavaScript
+
+This is the `script.js` file present in your repl just below the `index.html` file. JavaScript handles the logic of your webpage.
+
+Here, you have to write a small amount of JavaScript code because Firebase has some pre-defined functions for Authentication, which you will use here, and also, you can check [Firebase Documentation on web authentication](https://firebase.google.com/docs/auth/web/start).
+
+So what you have to do is get the currently signed-in user you created on Firebase.
+
+In the `script.js` file, you need to add a fuction called [Get the currently signed-in user](https://firebase.google.com/docs/auth/web/manage-users#get_the_currently_signed-in_user) for that which is:
+
+```javascript
+firebase.auth().onAuthStateChanged(function(user) {
+  if (user) {
+    // User is signed in.
+  } else {
+    // No user is signed in.
+  }
+});
+```
+As you created two divisions in the HTML file, you need to display a division with ID `login_div` at the login page and a division with ID `user_div` after the login page or when the user is signed-in.
+
+So in the above code, some changes have to be made.
+
+Add below two lines of code in the `if` condition of the above code.
+
+```javascript
+document.getElementById("user_div").style.display = "block";
+document.getElementById("login_div").style.display = "none";
+```
+
+The code should look like this:
+
+```javascript
+firebase.auth().onAuthStateChanged(function(user) {
+  if (user) {
+    // User is signed in.
+    document.getElementById("user_div").style.display = "block";
+    document.getElementById("login_div").style.display = "none";
+  } else {
+    // No user is signed in.
+  }
+});
+```
+
+In the code above, `block` will display the division, and `none` will hide it.
+
+As you added this for a signed-in user, you also need the `else` part. _where user is not signed-in_ means the user is on the login page, so the code will be the same as the code above, but you need to swap `block` and `none` to display only `main_div` with ID `login_div`.
+
+Add this code to the `else` condition.
+
+```javascript
+document.getElementById("user_div").style.display = "none";
+document.getElementById("login_div").style.display = "block";
+```
+
+The code should look like:
+
+```javascript
+firebase.auth().onAuthStateChanged(function(user) {
+  if (user) {
+    // User is signed in.
+    document.getElementById("user_div").style.display = "block";
+    document.getElementById("login_div").style.display = "none";
+  } else {
+    // No user is signed in.
+    document.getElementById("user_div").style.display = "none";
+    document.getElementById("login_div").style.display = "block";
+  }
+});
+```
+Now, how it looks like:
+
+![with JS](https://cloud-e3wysrojq.vercel.app/c1.png)
+
+After completing this, you need to work on both the functions you assigned in HTML buttons `login()` and `logout()`.
+
+##### Function login()
```

Comment:
@tanishq-soni I still have to review the rest but this should be a good start.

Reply:
sure np!